.eslintrc.json

```diff
@@ -16,6 +16,10 @@
         "curly": ["error", "multi-line"],
         "import/no-unresolved": [2, { "commonjs": true }],
         "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
+        "linebreak-style": [
+            2,
+            "windows"
```

Comment:
Instead of adding this, you can update your VSCode to LF on the right bottom side of your editor